src/Nox.Ui.Blazor.Lib/Components/NoxTypes/ViewStreetAddress.razor.cs

```diff
@@ -0,0 +1,69 @@
+﻿using Microsoft.AspNetCore.Components;
+using Nox.Types;
+using Nox.Ui.Blazor.Lib.Models;
+
+namespace Nox.Ui.Blazor.Lib.Components.NoxTypes;
+
+public partial class ViewStreetAddress : ComponentBase
+{
+
+    #region Declarations
+
+    [Parameter]
+    public StreetAddressModel? StreetAddress { get; set; }
+
+    [Parameter]
+    public List<CountryModel>? CountrySelectionList { get; set; }
```

Comment:
sorry @DanielHodgeGH  can you update this one as well? I think we have a nox convention for List to be empty by default (not null).

Reply:
yeah no problem i'll change this on the next PR thanks